source/docs/casper/resources/index.md

```diff
@@ -1,8 +1,20 @@
 # Table of Contents
 
-- [Why Build On Casper](./build-on-casper/index.md)
-  - [Ecosystem Open-Source Software](./build-on-casper/casper-open-source-software.md)
-- [Quickstart](./quick-start.md)
-- [Tutorials](./tutorials/index.md)
-  - [Beginner Tutorials](./tutorials/beginner/index.md)
-  - [Advanced Tutorials](./tutorials/advanced/index.md)
+Introduction
+  - [Wallets and Block Explorers](./build-on-casper/introduction/#wallets)
```

Comment:
Shouldn't this link and the following links be of this structure instead:
./build-on-casper/introduction.md#wallets
instead of
./build-on-casper/introduction/#wallets

Reply:
I think you are right. Both seem to work but I decided to change it.